doc/source/whatsnew/v2.2.1.rst

```diff
@@ -35,6 +35,7 @@ Bug fixes
 - Fixed bug in :func:`pandas.api.interchange.from_dataframe` which was raising for empty inputs (:issue:`56700`)
 - Fixed bug in :func:`pandas.api.interchange.from_dataframe` which wasn't converting columns names to strings (:issue:`55069`)
 - Fixed bug in :meth:`DataFrame.__getitem__` for empty :class:`DataFrame` with Copy-on-Write enabled (:issue:`57130`)
+- Fixed bug in :func:`format_percentiles` related to unneccesary brackets around type annotation (:issue:`57248`)
```

Comment:
We typically do not add a whatsnew entry for type annotations (and I'm not sure whether this will go into 3.0)

Reply:
Okay, thanks. I did not know. I can then simply remove my last commit related to the whatsnew entry, right? Because then it also doesn't clash with 3.0

Reply:
Yes